app/controllers/websites_controller.rb

```diff
@@ -4,7 +4,7 @@ def create
       redirect_to root_url
```

Comment:
`@web` is not being used anywhere. Ensure that `Websites::Create.call` returns a website, or `nil` if it fails.